javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1131,6 +1131,16 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000);
     }
 
+    // if not declared, attribute is set to 'undefined'
+    // (or)
+    // if declared (ex: sameSite:"") and string length is zero, attribute is 'None' by default
+    if(typeof sameSite === "undefined" || sameSite.length===0) {
+      // Do noting. sameSite value will be none by default
+    } else if(sameSite!=="undefined" && (sameSite!=="Strict" && sameSite!=="Lax")) {
+      throw new error.InvalidArgumentError(
+          'Invalid sameSite cookie value "' + sameSite + '". It should be either "Lax" (or) "Strict" ');
```

Comment:
```suggestion
          `Invalid sameSite cookie value ${sameSite}. It should be either "Lax" (or) "Strict" `);
```

Reply:
Hi @christian-bromann,

I've updated code as suggested. Can you please review